main/Accordion/AccordionItem.tsx

```diff
@@ -92,23 +93,20 @@ const AccordionItem: FC<AccordionItemProps> = (props) => {
 
     const {height} = e.nativeEvent.layout;
 
-    bodyHeight.current = height;
+    setBodyHeight(height);
     setBodyMounted(true);
   };
 
-  const [collapsed, setCollapsed] = useState(collapseOnStart);
-
   const handlePress = (): void => {
     setCollapsed(!collapsed);
   };
 
   const startAnimation = useCallback(() => {
-    const rotateTargetValue = collapsed ? 0 : 1;
-    const dropDownTargetValue = collapsed ? 0 : bodyHeight.current;
```

Comment:
@hyochan 
I changed bodyHeight value React.useState to React.useRef for https://github.com/dooboolab/dooboo-ui/pull/108#discussion_r765427352.
`useRef` doesn’t notify you when its content changes, so i think re-rendering of initial by collapsed state was not working. 
After initial bodyHeight value is rendering to 0, the value is keeping to 0 on this function without the side effect so i think it could't guaranteed to change to measured value after initial mounted.

Reply:
@luke-hanwook Hello. Thanks for coming back for a great explanation. I really appreciate it! 💯